ambuild2/util.py

```diff
@@ -92,11 +92,12 @@ def WaitForProcess(process):
   return process.returncode
 
 def CreateProcess(argv, executable = None):
-  pargs = { 'args': argv }
-  pargs['stdout'] = subprocess.PIPE
-  pargs['stderr'] = subprocess.PIPE
-  if executable != None:
-    pargs['executable'] = executable
+  pargs = {
+    'args': argv,
+    'stdout': subprocess.PIPE,
+    'stderr': subprocess.PIPE,
+    'executable': executable,
```

Comment:
This is technically a functional change since 'executable' is now a key where it wasn't before. I doubt it matters here though.